### PR TITLE
Allow log_entries for success response

### PR DESCRIPTION
**Pull request by medius**
@IrvinFan

### CODE REVIEW

**Comment by IrvinFan** [comment]
Reviewing

File: app/models/spree/gateway/pay_pal_express.rb [hunk 1]
```diff
@@ -71,18 +71,20 @@ def purchase(amount, express_checkout, gateway_options={})
...
         express_checkout.update_column(:transaction_id, transaction_id)
         # This is rather hackish, required for payment/processing handle_response code.
-        Class.new do
-          def success?; true; end
+        class << pp_response
           def authorization; nil; end
-        end.new
+          def to_s
+            do_express_checkout_payment_response_details
+          end
```
Comment (IrvinFan) [inline app/models/spree/gateway/pay_pal_express.rb]: I don't think the `to_s` method is needed. Can you check that?

Comment (medius) [inline app/models/spree/gateway/pay_pal_express.rb]: Yes, we don't need it. I was trying something and forgot to save the file without it before commiting. I took it out.

**Comment by IrvinFan** [comment]
Looks fine. Working as expect. Merging

**Pull request merged**